client/packages/common/src/ui/icons/index.ts

```diff
@@ -63,6 +65,7 @@ export { SuppliersIcon } from './Suppliers';
 export { ThermometerIcon } from './Thermometer';
 export { ToolsIcon } from './Tools';
 export { TranslateIcon } from './Translate';
+export { BarChart2Icon as TrendingUpIcon } from './BarChart2';
```

Comment:
> I changed the icon, due to those weird prop/styling issues I mentioned in the chat. This one is called BarChart2: https://feathericons.com/?query=barchart-2

But you're renaming here?

Reply:
Oh, sorry I missed that. Fixed now: 393a89062